app/javascript/app/components/ndcs/ndcs-enhancements-viz/ndcs-enhancements-viz.js

```diff
@@ -79,7 +79,7 @@ class NDCSEnhancementsVizContainer extends PureComponent {
       let tooltipTxt;
       switch (indicator.locations[id].label_slug) {
         case 'submitted_2020':
-          tooltipTxt = `Submitted a 2020 NDC on ${dateIndicator.locations[id].value}.`;
+          tooltipTxt = `Submitted a 2020 NDC on ${dateIndicator.locations[id].value}.\n\n${statementIndicator.locations[id].value}`;
```

Comment:
We prefer to use elements instead of  \n to break lines.
This could be a div to create a line break as it's already inside a `<p>`. 

Maybe with a className if you want further styling. Adding JSX to a container file is not the best option but this specific function I think is more related to the presentation of the text so it could be moved to the component instead of the container.

Reply:
Hi Kevin! You could move getTooltipText to the component and there use some divs for each paragraph. If the content is not separated enough you could add some className with a margin-bottom so the different lines are more split apart. I'm not sure what do you mean by an example. Let me know if you need more info.